<html>
<head>
<title>Isoleucyl-tRNA Synthetase</title>

<meta name="description" content="Isoleucyl-tRNA synthetase (IleRS) is an enzyme that attaches the amino acid isoleucine to its cognate tRNA during protein synthesis">
<meta name="keywords" content="IleRS,IRS,aminoacyl-tRNA synthetase,aminoacyl-tRNA ligase,AARS,database,application,tool,visualization,protein,biochemistry,molecular biology,bioinformatics">

<link rel="icon" >
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="../../css/styles.css">

<script language='JavaScript' type='text/JavaScript' src="/js/aaRS.js"></script>
<script language='JavaScript' type='text/JavaScript' src="/js/util.js"></script>
<script type='text/javascript' src='/js/bio-pv/bio-pv.min.js'></script>

<script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>   
        
<script type="text/javascript"
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true
  }
});
</script>



 <script src="https://d3js.org/d3.v4.js"></script>
 <script  type='text/javascript' src="/js/trna.js"></script>


<script async src="https://www.googletagmanager.com/gtag/js?id=G-25G2X1FEYG"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-25G2X1FEYG');
</script>






<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>




<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS();


}




</script>

</head>
<body onload="init()">


	<div id="main">
<div class='indexMetadata'><br><br>

<!-- Content automatically pulled from readme for indexing purposes. This will be overwritten using javascript -->
<br><br>

<h1>Isoleucyl tRNA Synthetase</h1><br><br>


Isoleucyl-tRNA synthetase (IleRS) is an enzyme that plays a crucial role in protein synthesis by catalyzing the attachment of the amino acid isoleucine to its cognate tRNA:





$ \text{Ile} + \text{tRNA}^\text{Ile} + \text{ATP} \xrightarrow{\text{IleRS}} \text{Ile-tRNA}^\text{Ile} + \text{AMP} + \text{PP}_i  $


IleRS features a N-terminal catalytic domain, C-terminal tRNA binding domains, and a post-transfer editing domain inserted within the catalytic domain (Nordin and Schimmel 2005).



IleRS is closely related to the [LeuRS-A](/class1/leu2), [LeuRS-B](/class1/leu1), [ValRS](/class1/val), and [MetRS](/class1/met) families, which comprise 
subclass Ia (Perona and Hadd 2012, Gomez and Ibba, 2020).
Members of subclass Ia are characterized by their hydrophobic amino acid substrates, as well as the connecting peptide 2 (CP2) and zinc finger (ZF) insertions, depicted below. 
CP2 is a 30-40 amino acid two-helix bundle on the surface of the catalytic domain (Starzyk et al. 1987) and appears to be essential for amino acid activation (Zhou at al. 2008). 
Upstream from this resides a cysteine-rich zinc finger, 20-40 amino acids in length, also essential for effective aminoacylation (Nureki et al. 1993, Sugiura et al. 2000). 


IleRS, LeuRS, and ValRS share a a post-transfer [editing domain](/superfamily/class1/Editing_domain_1a), absent from MetRS.
The editing domain typically resides within the zinc finger, providing further amino acid selectivity by expelling a wide range of mistargetted amino acids 
such alanine, cysteine, threonine, valine, isoleucine, methionine, homocysteine, and norvaline (Gomez and Ibba, 2020).
IleRS, ValRS, and the bacterial-like LeuRS-B are further characterized by connecting peptide 3 (CP3), which contains a second zinc-finger motif in certain organisms (Fukunaga et al. 2005).
Its functional role is unclear. 


There are two types of IleRS, which have distinct sequences and C-terminal domain architectures. 
Both forms possess a  [helical domain](/superfamily/class1/Anticodon_binding_domain_CRIMVL), that recognises the anticodon loop, residing downstream from the catalytic domain.
The domain is universal across the members of subclass Ia, as well as [CysRS](/class1/cys) and [ArgRS](/class1/arg).
However, the remaining C-terminal regions vary between the two types.
The first type, IleRS-1, is found in certain bacteria, organelles, and eukaryotic cytosols.
It possesses a [C-terminal junction domain](/superfamily/class1/C-terminal_junction_domain) and a zinc finger downstream from the helical bundle.
Together with an N-terminal extension, the three C-terminal domains recognise the anticodon loop (Silvian et al. 1999).  
The second type, IleRS-2, is found in many archaea, bacteria, and eukaryotic cytosols. 
It lacks the N-terminal extension and zinc finger motif (Brkic et al. 2023), and achieves anticodon loop recognition through a much larger variation of the C-terminal junction domain.






Many bacteria express both types of IleRS. Certain bacteria, such as *Staphylococcus aureus*, have an abnormal GxHH sequence profile in place of the standard HxGH motif in IleRS-2, promoting antibiotic resistance (Brkic et al. 2023).

<br><br>

<h2>References</h2><br><br>

Douglas, J, Bouckaert, R., Carter, C., & Wills, P. R. Enzymic recognition of amino acids drove the evolution of primordial genetic codes. Research Square (2023).



Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." Rna 26.8 (2020): 910-936.




Brian E. Nordin and Paul Schimmel. "The Aminoacyl-tRNA Synthetases" CRC Press (2005): Chapter 4: Isoleuyl-tRNA Synthetases.




Zhou, Xiao-Long, Bin Zhu, and En-Duo Wang. "The CP2 domain of leucyl-tRNA synthetase is crucial for amino acid activation and post-transfer editing." Journal of Biological Chemistry 283.52 (2008): 36608-36616.



Starzyk, Ruth M., Teresa A. Webster, and Paul Schimmel. "Evidence for dispensable sequences inserted into a nucleotide fold." Science 237.4822 (1987): 1614-1618.




Perona, John J., and Andrew Hadd. "Structural diversity and protein engineering of the aminoacyl-tRNA synthetases." Biochemistry 51.44 (2012): 8705-8729.



Nureki, O., et al. "Chemical modification and mutagenesis studies on zinc binding of aminoacyl-tRNA synthetases." Journal of Biological Chemistry 268.21 (1993): 15368-15373.




Sugiura, Ikuko, et al. "The 2.0 Ã… crystal structure of Thermus thermophilus methionyl-tRNA synthetase reveals two RNA-binding modules." Structure 8.2 (2000): 197-208.



Fukunaga, Ryuya, and Shigeyuki Yokoyama. "Crystal structure of leucyl-tRNA synthetase from the archaeon Pyrococcus horikoshii reveals a novel editing domain orientation." Journal of molecular biology 346.1 (2005): 57-71.



Nureki, Osamu, et al. "Enzyme structure with two catalytic sites for double-sieve selection of substrate." Science 280.5363 (1998): 578-582.



Silvian, Laura F., Jimin Wang, and Thomas A. Steitz. "Insights into editing from an ile-tRNA synthetase structure with tRNAile and mupirocin." Science 285.5430 (1999): 1074-1077.



Chen, Bingyi, et al. "Inhibitory mechanism of reveromycin A at the tRNA binding site of a class I synthetase." Nature communications 12.1 (2021): 1616.



Brkic, Alojzije, et al. "Antibiotic hyper-resistance in a class I aminoacyl-tRNA synthetase with altered active site signature motif." bioRxiv (2023): 2023-01.<br><br>

</div>
		</div>
		</body>
	</html>
