
The archaeal-like prolyl-tRNA synthetase (ProRS-A) is an enzyme that plays a crucial role in protein synthesis by catalyzing the attachment of the amino acid proline to its cognate tRNA:




$ \text{Pro} + \text{tRNA}^\text{Pro} + \text{ATP} \xrightarrow{\text{ProRS-A}} \text{Pro-tRNA}^\text{Pro} + \text{AMP} + \text{PP}_i  $






## Structure

The archaeal-like ProRS (ProRS-A)  consists of a conserved aminoacylation domain containing the three conserved motifs of Class II AARS. Like the bacterial-type ProRS [ProRS-B](/class2/pro2), ProRS-A has a dedicated binding domain for tRNA anticodon recognition. Additionally, ProRS-A encodes a C-terminal zinc-binding domain of ~80-110 amino acids in length. The functional activity of the zinc-binding domain remains unclear, but it appears to help maintain the protein’s tertiary structure and is essential for ProRS catalytic activity (Vasu et al. 2021). While ProRS-A is the predominant isoform in archaea and eukarya, it is also encoded by some bacteria (Crepin et al. 2006)

All ProRS types belong to subclass IIa, which includes other enzymes such as [ThrRS](/class2/thr), [SerRS](/class2/ser1), and the dimeric [GlyRS](/class2/gly1) (Gomez and Ibba 2020; Valencia-Sánchez et al. 2016; Perona et al. 2012). The [anticodon binding domains](/superfamily/class2/Anticodon_binding_domain_HGPT) of subclass IIa, except for SerRS, are homologous with that of [HisRS](/class2/his) and located at the C-terminal end (Wolf et al. 1999). The catalytic domain of ProRS is typical of a Class II aminoacyl-tRNA synthetase. Like most members of the superfamily, ATP binding is coordinated by the arginine tweezers located in motifs 2 and 3 (Kaiser et al. 2018).



<img src="/fig/ProRS_domain_architectures.png" alt="ProRS domain architecture"/>


In most animals, including humans, prolyl-tRNA is produced by a fusion protein glutamyl-prolyl tRNA synthetase (EPRS) (Berthonneau et al. 2000). 
EPRS contains the catalytic domains from both [GluRS-E](/class1/glu3/) and ProRS-A, as well as the ProRS-A zinc binding domain, and the GluRS GST domain. 
Glutamic acid is a metabolic precursor for proline, which suggests a possible explanation for the origin of this protein (Eswarappa et al. 2018). 



## tRNA recognition

ProRSs identify its tRNA substrates by recognition of the Pro anticodon, specifically bases G35 and G36 (Burke et al. 2001; Stehlin et al. 1998; Liu et al. 1995). ProRS-A from archaea and eukaryotes also moderately rely on the recognition of bases at positions 72 and 73, which correspond to C72 and A73 in archaeal tRNA<sup>Pro</sup> and C72 and C73 in cytosolic eukaryotic tRNA<sup>Pro</sup> (Burke et al. 2001; Stehlin et al. 1998). Bacterial ProRS-A recognizes bacterial-type tRNA<sup>Pro</sup>, which consists of G72 and A73. 


## Editing

ProRSs are inherently prone to ligating alanine to its cognate tRNA. Because the synthesis of Ala-tRNA<sup>Pro</sup> leads to the incorrect translation of proline codons with Ala, some ProRSs have acquired a hydrolytic domain (or editing domain) that catalyzes the deacylation of Ala-tRNA<sup>Pro</sup>, preventing accumulation (Beuning and Musier-Forsyth, 2000). In bacterial-like ProRS, the editing domain is inserted between the conserved Class II motifs 2 and 3 in the aminoacylation domain. Consequently, the bacterial ProRS editing is known as the insertion (INS) domain. This domain is not found in archaeal ProRSs. In contrast, ProRSs from some lower eukaryotes contain an editing domain. The editing domain of eukaryotic ProRS is at the N-terminus of the enzyme. However, in some cases, the domain is not functional, as observed for *Saccharomyces cerevisiae* ProRS (Ahel et al. 2003; SternJohn et al. 2007). The editing domain is functional in other organisms, such as *Plasmodium falciparum* (Ahel et al. 2003). Notably, the editing domain of *P. falciparum* ProRS is not directly related to the bacterial INS domain but is related to the ProXp-ala family (Ahel et al. 2003; Vargas-Rodriguez et al. 2020).

The ProXp-ala family is a member of the INS superfamily of aminoacyl-tRNA deacylases, including the INS domain of ProRS. ProXp-ala is more commonly found as single-domain enzymes than fused to ProRS, and it is widely distributed in bacteria and eukaryotes, including mammals, birds, and plants (Vargas-Rodriguez et al. 2013; Vargas-Rodriguez et al. 2020). Thus, ProXp-ala may compensate for the existence of ProRSs without an INS domain in many organisms. In addition to the ProXp-ala and INS domain families, the INS superfamily includes ProXp-x (Abu-tRNA deacylase), YbaK (Cys-tRNA deacylase), ProXp-ST1 (Ser/Thr-tRNA deacylase), ProXp-ST2 (Ser/Thr-tRNA deacylase), ProXp-7 (unknown), ProXp-8 (unknown), and ProXp-9 (unknown) (Kuzmishin Nagy et al. 2020). 


## Non-translational functions

In human cells, EPRS is part of the interferon (IFN)-gamma-activated inhibitor of translation (GAIT) complex involved in the translational silencing of genes involved in inflammation (Sampath et al., 2004). It is also a mTOR-S6K1 effector that regulates long-chain fatty acid uptake (Arif et al., 2017), and it is involved in organelle-specific inflammation resolution and antiviral immune function (Lee et al. 2024). All these noncanonical functions of EPRS are regulated by posttranslational modification of the WHEP domains of EPRS (Jia et al., 2008). In mammals, EPRS is part of the Multi Synthetase Complex (MSC), formed by several aaRSs.




## Clinical significance

The human EPRS gene is involved in autosomal recessive hypomyelinating leukodystrophy 15 (HLD15) caused by homozygous or compound heterozygous mutations (Mendes et al. 2018; Jiang et al. 2020).


