

# Organisms of AARS Online

The AARS operate the genetic code in the two-or-three domains of life - bacteria, archaea, and eukaryotes - as well as mitochondria and chloroplasts, and even in certain viruses.
AARS Online contains samples from these five genetic environments. Within each environment, the AlphaFold samples are taxonomically diverse and there are no two samples from the same phylum.
All sequences here are linked to their source data on GenBank (for AlphaFold predictions) or the Protein Data Bank (solved structures).



## <img class="imgIcon" src="/fig/Bacteria.png" alt="bacteria icon" />  Bacteria  

The Bacteria are a diverse domain of single-cellular organisms. 
There are many AARS that are exclusive, or largely restricted, to the bacteria, 
including the tetrameric [GlyRS](/class2/gly2), the editing-active [ProRS](/class2/pro2), and the forms of [GluRS](/class1/glu1) that have helical anticodon binding domains.





## <img class="imgIcon" src="/fig/Archaea.png" alt="archaea icon" /> Archaea  

The Archaea are primitive lifeforms that offer a glimpse into early history on our planet.
They are single-celled organisms that resemble bacteria, however their genes are more closely related to eukaryotes.
Like bacteria, they show high levels of horizontal gene transfer (Eppley et al. 2007, Papke et al. 2007).
Their taxonomy is still an active area of research, and therefore the Archaeal phyla are provisional.
There are several AARS that are exclusive, or largely restricted, to the Archaea, including [SepRS](/class2/sep), a special form of [SerRS](/class2/ser2), and the non-discriminating [GlxRS](/class1/glu2).


## <img class="imgIcon" src="/fig/Eukaryota.png" alt="leaf icon" /> Eukaryota 

The eukaryotes are organisms whose cells have a nucleus. This includes animals, plants, and fungi.
The eukaryotes are also characterized by gene splicing and the presence of mitochondria/chloroplasts, both of which can present problems when identifying and classifying protein sequences from genomic data. 
It is noted that some eukaryotic protein sequences are computational predictions that are not directly supported by mRNA sequencing data, in which case they should be considered as putative.


## <img class="imgIcon" src="/fig/Mitochondrial.png" alt="mitochondria icon" /> Mitochondria and chloroplasts 

Mitochondria and chloroplasts have their own translational machinery, distinct from the cytosols they reside in. 
They also have their own AARS, but these are not encoded by the small compartmental genomes. 
Rather, the enzymes are produced in the cytosol and are imported into the mitochondria and chloroplasts using localization signals.
In general, the same AARS is localized into both compartments through dual targeting (DuchÃªne 2005).
In the case of [AlaRS](/class2/ala), [GlyRS](/class2/gly3), [HisRS](/class2/his), and [ValRS](/class1/val), the same AARS operates in the cytoplasm and mitochondria, through alternative initiation (Chatton et al. 1986).
Due to the endosymbiotic origins of these organelles, their AARS are typically, but not always, of bacterial origin. 


Predicting protein cellular localization is a non-trivial problem and existing algorithms are not always accurate (Sun and Habermann 2017). 
Thus, the classifications of eukaryotic structures into organellar/cytosolic should be considered as putative unless there is direct experimental evidence. 
If you believe we have misclassified a eukaryotic structure into organellar/cytosolic, please contact us or raise an issue on GitHub.




## <img class="imgIcon" src="/fig/Viruses.png" alt="virus icon" />  Viruses 

Some giant viruses contain genes for expressing AARS.
The giant viruses typically infect amoeba, and have large double-stranded DNA genomes that encode around 1000 genes (Legendre et al. 2012).
Among these genes are the AARS, including  [AsnRS](/class2/asn), [ArgRS](/class1/arg), [CysRS](/class1/cys), [IleRS](/class1/ile), [MetRS](/class1/met),  [TrpRS](/class1/trp), and [TyrRS](/class1/tyr) (Arslan et al. 2011).
The emergence of giant viruses is an enigma that challenges the classical understanding of virus evolution (Koonin et al. 2018).  


