
<html>
<head>
<title>AARS Online</title>
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />

<meta name="description" content="AARS Online: an interative database of aminoacyl-tRNA synthetases">
<meta name="keywords" content="aminoacyl-tRNA synthetase,AARS,database,application,tool,visualization,protein,biochemistry,molecular biology,bioinformatics">


<link rel="icon" href="fig/icon.png">
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="css/styles.css">
<script language='JavaScript' type='text/JavaScript' src="/js/util.js"></script>
<!--
<script language='JavaScript' type='text/JavaScript' src="/js/mainPage.js"></script>
-->
        
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>

<script async src="https://www.googletagmanager.com/gtag/js?id=G-25G2X1FEYG"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-25G2X1FEYG');
</script>



<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>




<script language='JavaScript' type='text/JavaScript'>


function init(){

	renderHeader(false);



	// Add loading wheel
	//$("#catalyticDomainDIV").before(`<div id="mainloader" class='loader'><img src='/fig/loader.png'></img></div>`);
	//$("#catalyticDomainDIV").hide(0);


	//setTimeout(function(){


		//drawTree();

		// Remove loading wheel
		//$("#mainloader").remove();
		//$("#catalyticDomainDIV").show(200);



		//let eles = $(".aarsShowcase");
		//showEle(eles, 0);
		

	//}, 50);


	
}



showEle = function(eles, i, timeout = 200){



	if (i >= eles.length) return;
	
	let ele = $(eles[i]);
	ele.slideDown(100);

	setTimeout(function(){
		showEle(eles, i+1, timeout);
	}, timeout);
	


}



</script>

</head>
<body onload="init()">



	<div id="main">



		<div class="banner">


			<h1>Welcome to AARS Online</h1>


			The aminoacyl-tRNA synthetases (AARS) are a large group of enzymes that attach amino acids to their cognate tRNA. 
			They operate the genetic code in all living systems across all three domains of life- the bacteria, archaea, and eukaryota - plus mitochondria and chloroplasts, and there are even certain viruses that have AARS genes.
			The evolution of <a href="class1">Class I</a> and <a href="class2">Class II</a> AARS can be traced back earlier than the last universal common ancestor 3-4 billion years ago.


			AARS Online is a curated database showcasing a taxonomically diverse selection of AARS sequences and structures.



		<ul class="flexContainer_left">


			
			<li class="aarsShowcase">
				<span class="">
						Written and maintained by the AARS research community.
				</span>
			</li>

			
			<li class="aarsShowcase">
				<span class="">
						Enzymes from 
						Bacteria <img class="imgIcon" src="/fig/Bacteria.png"/>, 
						Archaea <img class="imgIcon" src="/fig/Archaea.png"/>, 
						Eukaryotes <img class="imgIcon" src="/fig/Eukaryota.png"/>, 
						Organelles <img class="imgIcon" src="/fig/Mitochondrial.png"/>, and 
						Viruses <img class="imgIcon" src="/fig/Viruses.png"/> 
				</span>
			</li>


			<li class="aarsShowcase">
				<span class="">
						Contains protein structures with direct experimental support 
						<img class="imgIcon" src="/fig/xray.png"/>
						and AlphaFold models <img class="imgIcon" src="/fig/alphafold.png"/>
				</span>
			</li>


			<li class="aarsShowcase">
				<span>
						Seamlessly navigate between protein sequences and structures with a user-friendly interface
				</span>
			</li>


			<li class="aarsShowcase">
				<span>
						Download curated multiple sequence alignments and phylogenetic trees
				</span>
			</li>



			<li class="aarsShowcase">
				<span class="">
						Start a discussion or report an issue on GitHub using the 'Discuss' link at the bottom of each page
				</span>
			</li>


	
		</ul>




		


		</div>
		
<!--
	<div id="contents">

		<h2>Contents</h1>

		<div>
			<a href="class1">Class I AARS</a>
		</div>
		<div>
			<a href="class2">Class II AARS</a>
		</div>


	</div>
-->

	<div id='catalyticDomainDIV' style="width:100%; text-align:center; overflow-x: auto;"></div>



	<div class="banner">




				<h1>Curation team</h1>

				<ul class="flexContainer_left2">

					<li class="aarsShowcase">
					
						<table>
							<tr>
								<td>
									<h3>Jordan Douglas</h3> 
								</td>
								<td>
										Administrator
								</td>
							</tr>
						</table>
						

						<div>
							<img src="https://profiles.auckland.ac.nz/jordan-douglas/photo">
							Jordan Douglas is a Research Fellow at the University of Auckland, New Zealand.	
							His research interests lie in phylogenetics, virus evolution, and the origin of genetic coding through the lens of the aminoacyl-tRNA synthetases.
							<a href="https://profiles.auckland.ac.nz/jordan-douglas">View profile.</a>
						</div>

					</li>


					<li class="aarsShowcase">

							<table>
							<tr>
								<td>
									<h3>Peter Wills</h3> 
								</td>
								<td>
										Curator
								</td>
							</tr>
						</table>
					
						<div>
							<img src="fig/profiles/peter.jpg">
							Peter Wills is an emeritus theoretical biologist at the University of Auckland.  His research focuses on the origin of genetic coding as a reflexive interplay between processes of chemistry and computation, each guiding the other through levels of increasing complexity and specificity, still visible in the phylogeny of AARS.
						</div>

					</li>


					<li class="aarsShowcase">
					

							<table>
							<tr>
								<td>
									<h3>Charles Carter Jr.</h3> 
								</td>
								<td>
										Curator
								</td>
							</tr>
						</table>

						<div>
							<img src="fig/profiles/charlie.png">
							Charlie Carter studied TrpRS mechanisms based on crystal structures. His studies of AARS urzymes—models for the ancestral AARS that first enforced the genetic code—helped to validate the Rodin and Ohno bidirectional coding hypothesis. He and Peter Wills then showed how base-paired AARS ancestry also could explain the substrate binding differences needed for a binary coding alphabet.
						</div>

					</li>


					<li class="aarsShowcase">
					

						<table>
							<tr>
								<td>
									<h3>Marie Sissler</h3> 
								</td>
								<td>
										Curator
								</td>
							</tr>
						</table>

						<div>
							<img src="/fig/profiles/marie.jpg">
							Marie Sissler is a Research Director at CNRS - Strasbourg University, France. 
							She works on diverse aspects of mitochondrial RNA biology. 
							She is particularly interested in mitochondrial aminoacyl-tRNA synthetases in diverse biological models such as humans and trypanosomatids, under healthy and pathological contexts.
							<a href="https://mito.unistra.fr/">Lab website.</a>
						</div>

					</li>





					<li class="aarsShowcase">
					

						<table>
							<tr>
								<td>
									<h3>Oscar Vargas-Rodriguez</h3> 
								</td>
								<td>
										Curator
								</td>
							</tr>
						</table>

						<div>
							<img src="/fig/profiles/oscar.jpg">
							Oscar Vargas-Rodriguez is an Assistant Professor at the University of Connecticut School of Medicine, USA. His research focuses on understanding how aminoacyl-tRNA synthetases and tRNAs regulate translational fidelity and its interplay with cellular homeostasis.
							<a href="https://health.uconn.edu/vargas-rodriguez-lab/">Lab website.</a>
						</div>

					</li>


					

					<li class="aarsShowcase">
					

						<table>
							<tr>
								<td>
									<h3>Henna Tyynismaa</h3> 
								</td>
								<td>
										Curator
								</td>
							</tr>
						</table>

						<div>
							<img src="/fig/profiles/henna.jpg">
							Henna Tyynismaa is a Professor in medical molecular genetics at the University of Helsinki, Finland. She is interested in the role of aminoacyl-tRNA synthetases in human diseases.
							<a href="https://www.helsinki.fi/en/researchgroups/medical-neurogenetics-group">Lab website.</a>
						</div>

					</li>




						<li class="aarsShowcase">
					

						<table>
							<tr>
								<td>
									<h3>John Perona</h3> 
								</td>
								<td>
										Curator
								</td>
							</tr>
						</table>

						<div>
							<img src="fig/profiles/john.jpg">
							John Perona served as Professor of Biochemistry at UCSB (1994-2011) and PSU (2011-2024). John writes the Earthward newsletter on Substack and is author of From Knowledge to Power: The Comprehensive Handbook of Climate Science and Advocacy. He investigates how advances in science and technology can be incorporated into policymaking, focusing on climate change and the transition to a renewable energy economy.
							<a href="https://www.fromknowledgetopower.com">View website.</a>
						</div>

					</li>

					<li class="aarsShowcase">
					

						<table>
							<tr>
								<td>
									<h3>Haissi Cui</h3> 
								</td>
								<td>
										Curator
								</td>
							</tr>
						</table>

						<div>
							<img src="/fig/profiles/haissi.jpg">
							The Cui lab studies mammalian aminoacyl-tRNA synthetases. They strive to explore how the localization of aminoacylation is controlled by developing chemical tools, employing biochemical methods, and studying cellular and animal models. Haissi's favorite aminoacyl-tRNA synthetase is ArgRS.
							<a href="https://haissicui-lab.ca">Lab website.</a>
						</div>

					</li>


					<li class="aarsShowcase">
					

						<table>
							<tr>
								<td>
									<h3>Frederic Fischer</h3> 
								</td>
								<td>
										Curator
								</td>
							</tr>
						</table>

						<div>
							<img src="/fig/profiles/fred.jpg">
							Frederic Fischer is an Associate Professor in biochemistry and molecular biology at the University of Strasbourg, France. He is interested in non-canonical utilizations of aminoacyl-tRNAs and in particular in aminoacyl-tRNA transferases that reroute them away from protein synthesis to introduce aminoacyl modifications in various macromolecules in prokaryotes and eukaryotes.
							<a href="https://dyps.unistra.fr/en/">Lab website.</a>
						</div>

					</li>


					<li class="aarsShowcase">
					

						<table>
							<tr>
								<td>
									<h3>Michael Ibba</h3> 
								</td>
								<td>
										Curator
								</td>
							</tr>
						</table>

						<div>
							<img src="fig/profiles/mike.png">
							The Ibba lab focuses on understanding the mechanisms that determine how cells ensure the accurate translation of the genetic code, 
							and how changes in the underlying processes impact cellular health and contribute to microbial pathogenesis and antibiotic persistence.
							<a href="https://sites.chapman.edu/ibbalab/">Lab website.</a>
						</div>

					</li>
					
					<li class="aarsShowcase">
					

						<table>
							<tr>
								<td>
									<h3>Claudia Alvarez Carreño</h3> 
								</td>
								<td>
										Curator
								</td>
							</tr>
						</table>

						<div>
							<img src="fig/profiles/claudia.jpg">
							Claudia Alvarez Carreño is a Newton International Fellow at University College London. 
							Claudia investigates the evolution of ancient proteins using bioinformatics.
							Her research seeks to uncover deep relationships among proteins involved in the central dogma of molecular biology.
							<a href="https://profiles.ucl.ac.uk/97724-claudia-alvarez-carreno">View profile.</a>
						</div>

					</li>

						
<!-- 

					<li class="aarsShowcase">
					

						<table>
							<tr>
								<td>
									<h3>Hubert Becker</h3> 
								</td>
								<td>
										Curator
								</td>
							</tr>
						</table>

						<div>
							<img src="https://www.shutterstock.com/image-vector/blank-avatar-photo-place-holder-600nw-1095249842.jpg">
							Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
							Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
						</div>

					</li>


					


					<li class="aarsShowcase">
					

						<table>
							<tr>
								<td>
									<h3>Lluís Ribas de Pouplana</h3> 
								</td>
								<td>
										Curator
								</td>
							</tr>
						</table>

						<div>
							<img src="https://www.shutterstock.com/image-vector/blank-avatar-photo-place-holder-600nw-1095249842.jpg">
							Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
							Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
						</div>

					</li>
 -->


				</ul>


				More profiles coming soon...


	</div>




	<div class="banner">




				<h1>Related databases</h1>

				<ul class="flexContainer_left">
					<li class="aarsShowcase">
					
						<h2><a target="_blank" href="http://trna.ucsc.edu/tRNAviz//summary/">tRNAviz</a></h2>

						<div>
							<i>
							Explore and visualize tRNA sequence features across the tree of life.
							</i>   <br><br>


							Lin, Brian Y., Patricia P. Chan, and Todd M. Lowe. "tRNAviz: explore and visualize tRNA sequence features." Nucleic acids research 47.W1 (2019): W542-W547.
							</div>

					</li>

					<li class="aarsShowcase">
						<h2><a target="_blank" href="http://bioinf.bio.uth.gr/aars/#/">Prokaryotic AARS Database</a></h2>
						<div>
							<i>
								A database of AARS sequence motifs.  
							</i><br><br>

							Chaliotis, Anargyros, Panayotis Vlastaridis, Dimitris Mossialos, Michael Ibba, Hubert D. Becker, Constantinos Stathopoulos, and Grigorios D. Amoutzias. "The complex evolutionary history of aminoacyl-tRNA synthetases." Nucleic acids research 45, no. 3 (2017): 1059-1068.
						</div>
					</li>

					<li class="aarsShowcase">
						<h2><a target="_blank" href="http://misynpat.org/">MiSynPat</a></h2>

						<div>
						<i>
							Mitochondrial Aminoacyl-tRNA Synthetases & Pathologies. 
							</i> <br><br>

							Moulinier, L., Ripp, R., Castillo, G., Poch, O., & Sissler, M. (2017). MiSynPat: An integrated knowledge base linking clinical, genetic, and structural data for disease‐causing mutations in human mitochondrial aminoacyl‐tRNA synthetases. Human mutation, 38(10), 1316-1324.
						</div>
					</li>
				</ul>


	</div>


	
		<div class="banner">


		

			<h1>Issues</h1>


			If you spot an error or have any inquiries, please create an issue or start a discussion on <a href="https://github.com/aarsonline/aarsonline.github.io">GitHub</a>. <br><br> 
			

			

	</div>


		<div class="banner">




			
				<h1>References</h1>

				A manuscript for this work is in preparation. In the meantime, if you find this website is helpful for your research please cite: <br><br>

				Douglas, J., Bouckaert, R., Carter Jr, C. W., & Wills, P. R. (2024). Enzymic recognition of amino acids drove the evolution of primordial genetic codes. Nucleic Acids Research, 52(2), 558-571.
			</div>


	</div>

</body>
</html>
