
Pyrrolysyl-tRNA synthetase (PylRS) is an enzyme that plays a crucial role in protein synthesis by catalyzing the attachment of the 22nd amino acid pyrrolysine to its cognate tRNA:




$ \text{Pyl} + \text{tRNA}^\text{Pyl} + \text{ATP} \xrightarrow{\text{PylRS}} \text{Pyl-tRNA}^\text{Pyl} + \text{AMP} + \text{PP}_i  $




PylRS is among the rarest AARS, found only in a small group of bacteria and archaea (Guo et al. 2022). Pyrrolysine is encoded by the stop codon UAG (Hao et al. 2002; Srinivasan et al. 2002), translated by a dedicated pyrrolysine tRNA (tRNA$^\text{Pyl}$). The activity of PylRS from Methanosarcina barkeri was first reported in 2004 (Polycarpo et al. 2004; Blight et al. 2004).


## Structure


PylRS exists in three different isoforms: PylSn-PylSc, dPylSn, and PylSn+PylSc (Guo et al. 2022). PylSn-PylSc is the predominant isoform in Archaea and consists of a single polypeptide with two domains: an N-terminal tRNA-binding domain (PylSn) and a C-terminal catalytic domain (PylSc). This PylRS isoform displays poor solubility (Suzuki et al. 2017) and behaves as a homodimer (Lee et al. 2008). In Bacteria, PylRS is generally encoded in two separate genes corresponding to the PylSn and PylSc domains, which yields the PylSn+PylSc isoform (Jiang et al. 2012). In vitro, PylSc alone is capable of tRNA aminoacylation, but PylSn is required in vivo (Herring et al. 2007). The fusion or fission of AARS genes that results in switching between homodimeric and heterotetrameric enzymes has also been observed in [AlaRS](/class2/ala) and [GlyRS-B](/class2/gly2). The third PylRS isoform (dPylSn) contains the PylSc only and is present primarily in archaeal species and a few bacteria (Borrel et al. 2014; Guo et al. 2022). 

The structure of PylRS's catalytic domain is similar to that of [HisRS](/class2/his), [SepRS](/class2/sep) and [PheRS](/class2/phe1), which are part of subclass IIc
(Kavran et al. 2007, Perona et al. 2012, Englert et al. 2013, Valencia-SÃ¡nchez et al. 2016, Douglas et al. 2023). However, its sequence closely resembles subclass IIb - [AsnRS](/class2/asn/), [AspRS](/class2/asp1), and [LysRS-II](/class2/lys/) (Fournier et al. 2009). Accordingly, PylRS is assigned to subclass IIe (Douglas et al. 2023).

Like AlaRS and SerRS, PylRS does not recognize the tRNA anticodon. Instead, PylRS selects its tRNA substrate by recognizing the base at position 73 and T-loop and variable loop elements (Suzuki et al. 2017). dPylSn recognizes the overall shape of tRNA$^\text{Pyl}$ (Krahn et al. 2023). The binding of ATP in the active site is coordinated by the arginine tweezers located in motifs 2 and 3 (Kaiser et al. 2018).


## Editing

No editing activity for PylRS has been reported (Gomez and Ibba, 2020). 

## Genetic code expansion

The promiscuous amino acid specificity, its large amino acid binding pocket, tRNA specificity, and orthogonality in bacteria and eukaryotes have made PylRS a powerful tool for expanding the genetic code to biosynthesize proteins with noncanonical amino acids (ncAAs) at desired positions (Wan et al. 2014). Over 100 different ncAAs have been installed in proteins using wild-type and engineered PylRSs.

## Non-translational functions

None reported.

## Clinical significance

None reported. Archaeal species encoding PylRS reside in the human gut.

