
This page describes three glutamyl-tRNA synthetase families, GluRS-B, GlxRS-B and GluGlnRS, that are found predominanly in bacteria.  These enzymes play a crucial role in protein synthesis by catalyzing the attachment of the amino acid glutamate to varying tRNA isoforms for the eventual incorporation of glutamate, or in some cases glutamine, correctly into proteins.

First, the discriminating bacterial-like glutamyl-tRNA synthetase (GluRS-B) attaches the amino acid glutamate to its cognate tRNA:


$ \text{Glu} + \text{tRNA}^\text{Glu} + \text{ATP} \xrightarrow{\text{GluRS-B}} \text{Glu-tRNA}^\text{Glu} + \text{AMP} + \text{PP}_i  $



Second, the non-discriminating bacterial-like glutamyl-tRNA synthetase (GlxRS-B) attaches the amino acid glutamate to tRNA isoforms that are code-cognate for the incorporation of either glutamate or glutamine into proeteins:
 


$ \text{Glu} + \text{tRNA}^\text{Glu} + \text{ATP} \xrightarrow{\text{GlxRS-B}} \text{Glu-tRNA}^\text{Glu} + \text{AMP} + \text{PP}_i  $ 


$ \text{Glu} + \text{tRNA}^\text{Gln} + \text{ATP} \xrightarrow{\text{GlxRS-B}} \text{Glu-tRNA}^\text{Gln} + \text{AMP} + \text{PP}_i  $

A further enzyme (see below) is required to convert text{Glu-tRNA}^\text{Gln} to text{Gln-tRNA}^\text{Gln} for correct incorporation of glutamine into proteins. 


Third, the bacterial-like  glutamyl-glutaminyl-tRNA synthetase (GluRSGlnRS) discriminates between Glu and Gln for the noncognate attachment of Glu to \text{tRNA}^\text{Gln}:


$ \text{Glu} + \text{tRNA}^\text{Gln} + \text{ATP} \xrightarrow{\text{GluGlnRS}} \text{Glu-tRNA}^\text{Gln} + \text{AMP} + \text{PP}_i  $ 

The product of this reaction also requires enzymic conversion of text{Glu-tRNA}^\text{Gln} to text{Gln-tRNA}^\text{Gln}. 


As discovered by Lamour and Mirande in 1994 and discussed more recently by Hadd and Perona 2014, the glutamyl- and glutaminyl-tRNA synthetases comprising subclass Ib (Lamour et al., 1994, Perona and Hadd 2012, Gomez et al., 2020), have a complex complex coevolutionary history. 
Their diversification occurred after the last universal common ancestor, with bacterial-like forms being characterized by an [$\alpha$-helical anticodon binding domain](/d/ek), and the archaeal and eukaryotic forms possessing a [$\beta$-barrel anticodon binding domain](/d/eq) (Satiecka et al., 1998).
Phylogeny strongly suggests that GlnRS originated in the eukaryota, and was only later acquired by certain bacteria, but never by archaea (Tumbula et al., 2000) through horizontal gene transfer (Lamour et al. 1994). Therefore, all eukaryotes express forms of both GlnRS and GluRS, while all archaea and more than 90% of bacterial species studied thus far do not possess a GlnRS-encoding gene. 



When GlxRS or GluGlnRS attaches Glu to the noncognate \text{tRNA}^\text{Gln}, a tRNA-dependent amidotransferase (AdT), replaces the Glu with a Gln residue, producing the \text{Gln-tRNA}^\text{Gln} required for correct incorporation of glutaminyl residues into proteins (Nirenberg and Wilcox, 1968, Sch√∂n et al., 1988, Curnow et al., 1997).

The transamidation reaction is:


$  \text{Glu-tRNA}^\text{Gln}  + \text{ATP} + \text{Gln} \xrightarrow{\text{GatCAB or GatDE AdT}} \text{Gln-tRNA}^\text{Gln} + \text{ADP} + \text{P}_i + \text{Glu}  $


The same enzyme corrects the attachment of Asp residues to \text{tRNA}^\text{Asn} by the nondiscriminating [AsxRS](/class2/asp2/)  (Becker et al., 1998, Raczniak et al. 2001).  This non-discriminating enzyme is found in all organisms that lack GlnRS, such as all archaea, which express [GlxRS-A](/class1/glu2/), as well as the majority of bacteria, which have a non-discriminating variant [GlxRS-B](/class1/glu1/) (Lapointe et al., 1986),  or more rarely the noncognate variant GluGlnRS which only attaches Glu to tRNA$^\text{Gln}$ and not to its cognate tRNA$^\text{Glu}$ (Skouloubris et al. 2003, Salazar et al. 2003).








## Structure


GluRS-B (Nureki et al., 1995) is the bacterial counterpart of the eurarchaeal [GluRS-E](/class1/glu3), and GlxRS-B (Schulze et al., 2006) and GluGlnRS are bacterial counterparts of the eurarchaeal GlxRS-A, as differentiated by these enzymes' distinct catalytic domain sequences and anticodon binding domain structures.
The three largely bacterial families are challenging to distinguish using sequence-based comparisons (Schulze et al. 2006), and hence their putative functional assignments
are unified in a single alignment below.



The N-terminal catalytic domains of GluRS-B, GlxRS-B, and GluGlnRS closely resemble the those of other members of subclass Ib: <a href="/class1/gln">GlnRS</a>, GlxRS-A, and GluRS-E.
They are characterized by an insertion within CP1, containing a loop flanked by two helices (SC1b IM), which 
may play a role in acceptor stem recognition  (Rath et al. 1998, Nureki et al. 2010).
The $\alpha$-helical anticodon binding domains of GluRS-B and GlxRS-B are located at their C-terminal ends and are homologous with that of [LysRS-I](/class1/lys/) (Terada et al. 2002), in contrast to the other members of Ib, which instead have a $\beta$-barrel anticodon binding domain of archaeal origin. 
The members of subclass Ib, alongside [ArgRS](/class1/arg/) and LysRS-I, require the presence of tRNA to catalyze activation of the amino acid substrate (Mitra and Mehler 1966, Ibba et al. 1999).







## Editing


## Non-translational functions



## Clinical significance



In the human mitochondria, Gln-tRNA$^\text{Gln}$ is produced through through the transmidation pathway, as in bacteria.  A non-discriminating mt-GlxRS generates Glu-tRNA$^\text{Gln}$ prior to its conversion by a mitochondrial GatCAB AdT into Gln-tRNA$^\text{Gln}$ (Nagao et al. 2009, Echevarria et al. 2014).
Biallelic recessive variants (homozygous and compound heterozygous) in the human gene EARS2, which encodes the mitochondrial GluRS, cause the brain disease Leukoencephalopathy (LTBL), which affects the thalamus and brainstem and results in high lactate (first described in Steenweg et al. 2012).   Mutations have been mapped to all domains of the mt-GluRS. An exhaustive list of known disease-causing variants of EARS2 can be found [here](http://misynpat.org/misynpat/PageMaker.rvt?name=EARS2). The list idengtifies the location of sequence and structural changes caused by mutations, as well as the degree of conservation of affected residues, and relevant references are provided. 

