


<html>
<head>
<title>Alanyl-tRNA Synthetase</title>

<meta name="description" content="Alanyl-tRNA synthetase (AlaRS) is an enzyme that attaches the amino acid alanine to its cognate tRNA during protein synthesis">
<meta name="keywords" content="AlaRS,ARS,aminoacyl-tRNA synthetase,AARS,database,application,tool,visualization,protein,biochemistry,molecular biology,bioinformatics">

<link rel="icon" >
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="../../css/styles.css">

<script language='JavaScript' type='text/JavaScript' src="/js/aaRS.js"></script>
<script language='JavaScript' type='text/JavaScript' src="/js/util.js"></script>
<script type='text/javascript' src='/js/bio-pv/bio-pv.min.js'></script>


        
<script type="text/javascript"
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true
  }
});
</script>



<script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>   

<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>


<script async src="https://www.googletagmanager.com/gtag/js?id=G-25G2X1FEYG"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-25G2X1FEYG');
</script>


<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS();
}



</script>

</head>
<body onload="init()">


	<div id="main">
<div class='indexMetadata'><br><br>

<!-- Content automatically pulled from readme for indexing purposes. This will be overwritten using javascript -->
<br><br>

<h1>Alanyl-tRNA Synthetase</h1><br><br>

Alanyl-tRNA synthetase (AlaRS) is an enzyme that plays a crucial role in protein synthesis by catalyzing the attachment of the amino acid alanine to its cognate tRNA:

  
  
  
  
  

$ \text{Ala} + \text{tRNA}^\text{Ala} + \text{ATP} \xrightarrow{\text{AlaRS}} \text{Ala-tRNA}^\text{Ala} + \text{AMP} + \text{PP}_i $

  
  

  
  

The catalytic core of AlaRS is similar to the bacterial-like [GlyRS-B](/class2/gly2), which together comprise subclass IId (Valencia-Sánchez et al. 2016). The alpha helix rich tRNA binding domain primarily recognises elements within the tRNA acceptor stem (Naganuma et al. 2009). Although the catalytic domain of AlaRS is one of the smallest in the superfamily, the full enzyme is quite large and contains a unique domain architecture. Homodimerization in AlaRS occurs via a C-terminal coiled coil, unlike most class II synthetases which dimerise at the catalytic core (Naganuma et al. 2009). However, in certain organisms such as *Nanoarchaeum equitans*, AlaRS is produced by two genes which express the N- and C-terminal regions respectively, and thus likely behaves as a heterotetramer in such systems (Arutaki et al. 2020). The fusion or fission of AARS genes that result in switching between homodimeric and heterotetrameric enzymes has also been observed in GlyRS-B and [PylRS](/class2/pyl).

  
 The catalytic domains of AlaRS and GlyRS-B are characterized by their absence of the small interface between motifs 1 and 2 (Douglas et al. 2023). The small interface promotes dimerisation in other Class II synthetases, and therefore  AlaRS and GlyRS-B form oligomers through alternative means.
However, like most Class II synthetases, ATP binding is coordinated by the arginine tweezers, located in motifs 2 and 3 (Kaiser et al. 2018). In bacteria and eukaryotes, the AlaRS catalytic domain is characterized by the 30-40 residue AlaRS insertion module between motifs 2 and 3. The module is stabilized by a short $\beta$ strand that runs parallel with the six stranded antiparallel fold of the catalytic domain.
This module is absent in archaea.

  

Editing in AlaRS occurs at the post-transfer level through an editing domain which expunges mischarged glycyl and seryl groups from tRNA$^\text{Ala}$ (Tsui et al. 1981, Guo et al. 2009b). AlaX proteins, which are homologous with the editing domain, also target Ser-tRNA$^\text{Ala}$ (Beebe et al. 2008). This double checkpoint system minimizes serine-for-alanine mistranslations, which are known to cause profound problems in nature (Guo et al. 2009a, Schimmel 2011). The C-ala domain is tethered to the editing domain through a coiled coil and promotes cooperative binding between the catalytic core and the editing domains to tRNA$^\text{Ala}$ (Guo et al. 2009b). The [editing domain](/superfamily/class2/Editing_domain_AT/) of AlaRS is downstream of the catalytic domain and is homologous with that of [ThrRS](/class2/thr/), which conversely resides upstream of the catalytic domain (Sankaranarayanan et al. 1999).

  
  
  

In eukaryotes, AlaRS is one of the few aminoacyl-tRNA synthetases which operate in both the cytoplasm and mitochondria, alongside [GlyRS](/class2/gly3), [HisRS](/class2/his/), and [ValRS](/class1/val/). The two forms of AlaRS were likely of organellar, and hence bacterial, origin. Therefore, the archaeal form of AlaRS stands apart from the rest of the family. Localisation into these compartments is achieved through alternative initiation, which governs the expression of an N-terminal mitochondrial localisation signal (Tang et al. 2004). One exception to this is the AlaRS of the yeast *Vanderwaltozyma polyspora*, which has two genes encoding AlaRS; one for either compartment (Chang et al. 2011). It is likely that these two genes arose from a recent duplication event.
<br><br>

<h2>References</h2><br><br>



Schimmel, Paul. "Mistranslation and its control by tRNA synthetases." Philosophical Transactions of the Royal Society B: Biological Sciences 366.1580 (2011): 2965-2971.



Guo, Min, et al. "Paradox of mistranslation of serine for alanine caused by AlaRS recognition dilemma." Nature 462.7274 (2009a): 808-812.



Guo, Min, et al. "The C-Ala domain brings together editing and aminoacylation functions on one tRNA." Science 325.5941 (2009b): 744-747.




Tang, Huei-Lin, et al. "Translation of a yeast mitochondrial tRNA synthetase initiated at redundant non-AUG codons." Journal of Biological Chemistry 279.48 (2004): 49656-49663.


Cavarelli, Jean, et al. "L-arginine recognition by yeast arginyl-tRNA synthetase." The EMBO journal 17.18 (1998): 5438-5448.

Naganuma, Masahiro, et al. "Unique protein architecture of alanyl-tRNA synthetase for aminoacylation, editing, and dimerization." Proceedings of the National Academy of Sciences 106.21 (2009): 8489-8494.



Chang, Chia-Pei, et al. "Alanyl-tRNA synthetase genes of Vanderwaltozyma polyspora arose from duplication of a dual-functional predecessor of mitochondrial origin." Nucleic acids research 40.1 (2012): 314-322.



Beebe, Kirk, et al. "Distinct domains of tRNA synthetase recognize the same base pair." Nature 451.7174 (2008): 90-93.



Sankaranarayanan, Rajan, et al. "The structure of threonyl-tRNA synthetase-tRNAThr complex enlightens its repressor activity and reveals an essential zinc ion in the active site." Cell 97.3 (1999): 371-381.



Tsui, W-C., and Alan R. Fersht. "Probing the principles of amino acid selection using the alanyl-tRNA synthetase from Escherichia coli." Nucleic Acids Research 9.18 (1981): 4627-4637.



Kaiser, Florian, et al. "Backbone brackets and arginine tweezers delineate class I and class II aminoacyl tRNA synthetases." PLoS computational biology 14.4 (2018): e1006101.




de Pouplana, Lluis Ribas, et al. "The Aminoacyl-tRNA Synthetases" CRC Press (2005): Chapter 21: Alanyl-tRNA Synthetases.



Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." Rna 26.8 (2020): 910-936.



Cusack, Stephen, Michael Härtlein, and Reuben Leberman. "Sequence, structural and evolutionary relationships between class 2 aminoacyl-tRNA synthetases." Nucleic acids research 19.13 (1991): 3489-3498.



Valencia-Sánchez, Marco Igor, et al. "Structural Insights into the Polyphyletic Origins of Glycyl tRNA Synthetases." Journal of Biological Chemistry 291.28 (2016): 14430-14446.

Douglas, J, Bouckaert, R., Carter, C., & Wills, P. R. Enzymic recognition of amino acids drove the evolution of primordial genetic codes. Research Square (2023).

Arutaki, Misa, et al. "G: U-Independent RNA minihelix aminoacylation by Nanoarchaeum equitans alanyl-tRNA synthetase: an insight into the evolution of aminoacyl-tRNA synthetases." Journal of molecular evolution 88 (2020): 501-509.
<br><br>

</div>
		</div>
		</body>
	</html>
