
Tryptophanyl-tRNA synthetase (TrpRS) is an enzyme that plays a crucial role in protein synthesis by catalyzing the attachment of tryptophan to its cognate tRNA:




$ \text{Trp} + \text{tRNA}^\text{Trp} + \text{ATP} \xrightarrow{\text{TrpRS}} \text{Trp-tRNA}^\text{Trp} + \text{AMP} + \text{PP}_i  $






## Structure 

TrpRS, together with its close homolog [TyrRS](/class1/tyr), form the subclass Ic. Consequently, TrpRS and TyrRS share several structural features and phylogenetic origins, possibly emerging from a common ancestor that may have had dual amino acid specificity (Doublié et al. 1995, Fournier et al. 2015). Extant TrpRS and TyrRS acquired specificity for their respective amino acid substrates via conserved aspartates located in the specificity-determining helix (SDH) (Doublié et al. 1995). Additionally, while most Class I synthetases approach the tRNA acceptor stem from the minor groove side, TyrRS and TrpRS approach the acceptor stem directly instead (de Pouplana and Schimmel 2001).

TrpRS from prokaryotic and archaeal/eukaryotic organisms form distinct groups with respect to their sequences, allostery, and domain architectures (Carter 2005). The prokaryotic TrpRS form is among the smallest AARSs in length, consisting of approximately 330 amino acids. Structurally, TrpRS displays a compact domain organization encompassing an aminoacylation domain with the canonical Class I motifs and a C-terminal anticodon binding domain [anticodon binding domain](/d/wy). Both domains are connected by the connecting peptide 1 (CP1) domain, which occurs in its simplest form in TrpRS and TyrRS. The CP1 mediates the homodimerization of both AARSs. The archaeal-type TrpRS contains an additional N-terminal domain known as the eukaryote-specific-extension (ESE) (Yang et al. 2007). Both the ESE and the C-terminal anticodon binding domain undergo vast conformational changes upon substrate binding, making the archaeal-like form quite complex (Shen et al. 2008). However, despite the intricate allostery displayed by TrpRS, the catalytic domain is less elaborate than other Class I synthetases, making it a useful scaffold for investigating ancestral models, such as urzymes (Pham et al. 2010; Onodera et al. 2021), which display activation and aminoacylation activity respectively. 

tRNA recognition by TrpRS is mediated by anticodon and acceptor stem interactions. However, tRNA acceptor stem identities have diverged from bacteria to archaea and eukaryotes, creating differences in tRNA recognition by the prokaryotic and archaeal/eukaryotic enzymes (Giegé and Eriani, 2023). Consequently, prokaryotic TrpRS does not cross-react with archaeal/eukaryotic tRNA and vice versa (Xu et al. 2001; Chatterjee et al. 2013).




## Editing

No editing activity has been reported for TrpRS. 


## Non-translational functions

Some eukaryotes contain an additional domain upstream of the ESE.
In humans, TrpRS contains the WHEP domain, which plays a role in the primary defense system against infection (Ahn et al. 2017). Human TrpRS is strongly upregulated upon IFN-gamma stimulation and can mediate high affinity tryptophan uptake (Miyanokoshi et al., 2018; Yokosawa et al., 2020). Human TrpRS can also form a complex with DNA-dependent protein kinase and PARP-1, which links IFN-gamma and p53 signaling (Sajish et al., 2012). 

In certain parasites, TrpRS possesses an N-terminal domain homologous to the editing domain of [ThrRS](/class2/thr) and [AlaRS](/class2/ala) (Merritt et al. 2011).

The rat mitochondrial TrpRS (WARS2) was shown to be a determinant for angiogenesis (Wang et al. 2016)


## Clinical significance

One variant of the human gene WARS1 which encodes the cytosolic TrpRS has been associated with Charcot-Marie-Tooth neuropathy (Tsai et al. 2017). 

Biallelic recessive variants (homozygous and compound heterozygous) in the human gene WARS2 encoding the mitochondrial TrpRS cause neurodevelopmental disorder with movement disorders (as early-onset tremor-parkinsonism syndrome) and intellectual disability (firstly described in Musante et al. 2017). The exhaustive list of disease-causing variants of mt-TrpRS (including location on the sequence and structure, the conservation of affected residues, and the related references) can be found [here](http://misynpat.org/misynpat/PageMaker.rvt?name=WARS2). Mutations have been mapped to all domain of the mt-TrpRS.



