The bacterial-like prolyl-tRNA synthetase (ProRS-B) is an enzyme that plays a crucial role in protein synthesis by catalyzing the attachment of the amino acid proline to its cognate tRNA:




$ \text{Pro} + \text{tRNA}^\text{Pro} + \text{ATP} \xrightarrow{\text{ProRS-B}} \text{Pro-tRNA}^\text{Pro} + \text{AMP} + \text{PP}_i  $




## Structure

The bacterial-like ProRS consists of a conserved aminoacylation domain containing the three conserved motifs of Class II AARS. All ProRSs have a dedicated binding domain for tRNA anticodon recognition. ProRS-B exists in two isoforms. One isoform has a domain (called the INS domain) for ~180 residues long inserted between the conversed Class II motifs 2 and 3. The second isoform encodes a truncated version of the INS domain, consisting of ~40 amino acids. Notably, the isoform with the truncated INS domain arose twice independently during evolution, leading to two distinct versions, one encoded in organelles and the other in bacteria. In bacteria, ProRS-B with the INS domain is the predominant isoform in bacteria whereas an estimated 11% of bacteria encode an Archaeal-like ProRS [ProRS-A](/class2/pro1) (Vargas-Rodriguez et al. 2013).

All ProRS types belong to subclass IIa, which includes other enzymes such as [ThrRS](/class2/thr), [SerRS](/class2/ser1), and the dimeric [GlyRS](/class2/gly1) (Gomez and Ibba 2020; Valencia-SÃ¡nchez et al. 2016; Perona et al. 2012). The [anticodon binding domains](/superfamily/class2/Anticodon_binding_domain_HGPT) of subclass IIa, except for SerRS, are homologous with that of [HisRS](/class2/his) and located at the C-terminal end (Wolf et al. 1999). The catalytic domain of ProRS is typical of a Class II aminoacyl-tRNA synthetase. Like most members of the superfamily, ATP binding is coordinated by the arginine tweezers located in motifs 2 and 3 (Kaiser et al. 2018).


<img src="/fig/ProRS_domain_architectures.png" alt="ProRS domain architecture"/>



In eukaryotic organelles, a specialized type ProRS-M is found. ProRS-M is one of the few AARS families lacking an experimentally solved structure. Phylogenetic analyses suggest that the enzyme was derived from the bacterial form ProRS-B, likely concurring with mitochondrial/chloroplast endosymbiosis (Douglas et al. 2024). Shortly after this event, the post-transfer editing domain was deleted in most members of ProRS-M. However, the N- and C-terminal regions flanking this excision site were retained, an evolutionary "scar". This scar is rich in highly conserved cysteine residues; its functional status is unclear.




## tRNA recognition

ProRSs identify their tRNA substrates by recognition of the Pro anticodon, specifically bases G35 and G36 (Burke et al. 2001; Stehlin et al. 1998; Liu et al. 1995). ProRS-B also strongly relies on recognizing bases at positions 72 and 73, corresponding to G72 and A73 in bacterial tRNA<sup>Pro</sup> (Liu et al. 1995).


## Editing

ProRS-B is inherently prone to ligating alanine to its cognate tRNA. Because the synthesis of Ala-tRNA<sup>Pro</sup> leads to the incorrect translation of proline codons with Ala, some ProRSs have acquired a hydrolytic domain (or editing domain) that catalyzes the deacylation of Ala-tRNA<sup>Pro</sup>, preventing accumulation (Beuning and Musier-Forsyth, 2000). The INS domain of ProRS-B carries out this function. This domain is not found in archaeal ProRSs. In contrast, ProRSs from some lower eukaryotes contain an editing domain. The editing domain of eukaryotic ProRS is in the N-terminus of the enzyme. However, in some cases, the domain is not functional, as observed for *Saccharomyces cerevisiae* ProRS (Ahel et al. 2003; SternJohn et al. 2007).  The editing domain is functional in other organisms, such as *Plasmodium falciparum* (Ahel et al. 2003). Notably, the editing domain of *P. falciparum* ProRS is not directly related to the bacterial INS domain but is related to the ProXp-ala family (Ahel et al. 2003; Vargas-Rodriguez et al. 2020). 

The INS domain of ProRS-B is part of the INS superfamily of aminoacyl-tRNA deacylases. The INS superfamily groups several other families, including the ProXp-ala, ProXp-x, ProXp-ST1, ProXp-ST2, and YbaK families (Kuzmishin Nagy et al. 2020). These enzymes are responsible for helping maintain translational fidelity during protein synthesis. The INS superfamily shares a conserved tertiary structure but displays diverse aminoacyl-tRNA substrate specificities. For example, ProXp-ala and the INS domain of ProRS hydrolyze Ala-tRNAPro, YbaK deacylates Cys-tRNAPro, and ProXp-ST1 and ProXp-ST2 hydrolyze Ser/Thr-tRNAs. Members of the INS superfamily are found in all domains of life, but each family has a distinct phylogenetic distribution (Kuzmishin Nagy et al. 2020); in general, they are distributed more predominantly in bacteria. 




## Non-translational functions



## Clinical significance

Biallelic recessive variants (homozygous and compound heterozygous) in the human gene PARS2 encoding the mitochondrial ProRS cause a disease named Alpers syndrome (firstly described in Sofou et al. 2015). The exhaustive list of disease-causing variants of PARS2 (including location on the sequence and the structure, the conservation fo the affected residues, and the related references) can be found [here](http://misynpat.org/misynpat/PageMaker.rvt?name=PARS2). Mutations have been mapped to all domains of the mt-ProRS except the N-terminal mitochondrial targeting sequence.