<html>
<head>
<title>Bacterial-like leucyl-tRNA Synthetase</title>
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=0.4" />
<meta name="description" content="The bacterial-like leucyl-tRNA synthetase (LeuRS-B) is an enzyme that attaches the amino acid leucine to its cognate tRNA during protein synthesis">
<meta name="keywords" content="bacteria,e coli,LeuRS,LRS,aminoacyl-tRNA synthetase,aminoacyl-tRNA ligase,AARS,database,application,tool,visualization,protein,biochemistry,molecular biology,bioinformatics">

<link rel="icon" >

<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="../../css/styles.css">

<script language='JavaScript' type='text/JavaScript' src="/js/aaRS.js"></script>
<script language='JavaScript' type='text/JavaScript' src="/js/util.js"></script>
<script type='text/javascript' src='/js/bio-pv/bio-pv.min.js'></script>

<script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>   
        
<script type="text/javascript"
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true
  }
});
</script>


 <script src="https://d3js.org/d3.v4.js"></script>
 <script  type='text/javascript' src="/js/trna.js"></script>



<script async src="https://www.googletagmanager.com/gtag/js?id=G-25G2X1FEYG"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-25G2X1FEYG');
</script>






<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>




<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS();


}




</script>

</head>
<body onload="init()">


	<div id="main">
<div class='indexMetadata'><br><br>

<!-- Content automatically pulled from readme for indexing purposes. This will be overwritten using javascript -->
<br><br>

<h1>Leucyl-tRNA Synthetase (Bacterial-like)</h1><br><br>



The bacterial-like leucyl-tRNA synthetase (LeuRS-B) is an enzyme that plays a crucial role in protein synthesis by catalyzing the attachment of the amino acid leucine to its cognate tRNA:




$ \text{Leu} + \text{tRNA}^\text{Leu} + \text{ATP} \xrightarrow{\text{LeuRS-B}} \text{Leu-tRNA}^\text{Leu} + \text{AMP} + \text{PP}_i  $




LeuRS-B is found in most bacteria and organelles, and differs from the archaeal-like form [LeuRS-A](/class1/leu2), which is found in most archaea. 
Either type may reside in a eukaryotic cytosol.   The two forms are characterized by their abilities to charge tRNAs corresponding to six different codons.
In the standard genetic code, [ArgRS](/class1/arg) and [SerRS](/class1/ser1) are the only other AARS which decode as many as six codons.
The structures of LeuRS-A and -B are distinguished by 
i) the placement of the editing domain within their primary sequence  (Fukunaga et al. 2005), 
ii) the C-terminal domain which recognises tRNA (Tukalo et al. 2005), and 
iii) the LeuRS-B insertion module below, also known as the leucyl-specific domain (Cusack et al. 2000).


LeuRS-A and -B alike are closely related to the  [IleRS](/class1/ile), [ValRS](/class1/val), and [MetRS](/class1/met) families, which comprise 
subclass Ia (Perona and Hadd 2012, Gomez and Ibba, 2020).
Members of subclass Ia are characterized by their hydrophobic amino acid substrates, as well as the connecting peptide 2 (CP2) and zinc finger (ZF) insertions, depicted below. 
CP2 is a 30-40 amino acid two-helix bundle on the surface of the catalytic domain (Starzyk et al. 1987) and appears to be essential for amino acid activation (Zhou et al. 2008). 
Upstream from this resides a cysteine-rich zinc finger, 20-40 amino acids in length, also essential for effective aminoacylation (Nureki et al. 1993, Sugiura et al. 2000). 


ValRS,	IleRS, and LeuRS share a post-transfer [editing domain](/superfamily/class1/Editing_domain_1a), absent from MetRS.
This editing domain typically resides within the zinc finger, providing further amino acid selectivity by expelling a wide range of mistargetted amino acids 
such alanine, cysteine, threonine, valine, isoleucine, methionine, homocysteine, and norvaline (Gomez and Ibba, 2020).
However, unlike LeuRS-A, ValRS, and IleRS, the editing domain of LeuRS-B resides downstream of the zinc finger and CP2 (Fukunaga et al. 2005).
Thus, the domain appears to have hopped between the two positions, or may have originated as a trans-acting factor.
IleRS, ValRS, and LeuRS-B are further characterized by connecting peptide 3 (CP3), which contains a second zinc-finger motif in certain organisms (Fukunaga et al. 2005).
Its functional role is unclear. 




The tRNA is recognised by two C-terminal domains (Tukalo et al. 2005, Rock et al. 2007).
The first is a  [helical domain](/superfamily/class1/Anticodon_binding_domain_CRIMVL) that recognises the anticodon binding domain, and is universal across the members of subclass Ia plus [CysRS](/class1/cys) and ArgRS.
The second is a compact $\alpha$-$\beta$ domain unique to LeuRS-B.
The domain is flexibly linked with the rest of the protein and mutational studies suggest it is essential for effective aminoacylation (Tukalo et al. 2005).


<br><br>

<h2>References</h2><br><br>

Douglas, J, Bouckaert, R., Carter, C., & Wills, P. R. Enzymic recognition of amino acids drove the evolution of primordial genetic codes. Research Square (2023).



Tommie L. Lincecum, Jr. and Susan A. Martinis. "The Aminoacyl-tRNA Synthetases" CRC Press (2005): Chapter 5: Leucyl-tRNA Synthetases.



Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." Rna 26.8 (2020): 910-936.



Cusack, Stephen, Anna Yaremchuk, and Michael Tukalo. "The 2 Å crystal structure of leucyl-tRNA synthetase and its complex with a leucyl-adenylate analogue." The EMBO Journal 19.10 (2000): 2351-2361.




Zhou, Xiao-Long, Bin Zhu, and En-Duo Wang. "The CP2 domain of leucyl-tRNA synthetase is crucial for amino acid activation and post-transfer editing." Journal of Biological Chemistry 283.52 (2008): 36608-36616.



Starzyk, Ruth M., Teresa A. Webster, and Paul Schimmel. "Evidence for dispensable sequences inserted into a nucleotide fold." Science 237.4822 (1987): 1614-1618.




Perona, John J., and Andrew Hadd. "Structural diversity and protein engineering of the aminoacyl-tRNA synthetases." Biochemistry 51.44 (2012): 8705-8729.




Nureki, O., et al. "Chemical modification and mutagenesis studies on zinc binding of aminoacyl-tRNA synthetases." Journal of Biological Chemistry 268.21 (1993): 15368-15373.





Sugiura, Ikuko, et al. "The 2.0 Å crystal structure of Thermus thermophilus methionyl-tRNA synthetase reveals two RNA-binding modules." Structure 8.2 (2000): 197-208.



Fukunaga, Ryuya, and Shigeyuki Yokoyama. "Crystal structure of leucyl-tRNA synthetase from the archaeon Pyrococcus horikoshii reveals a novel editing domain orientation." Journal of molecular biology 346.1 (2005): 57-71.



Rock, Fernando L., et al. "An antifungal agent inhibits an aminoacyl-tRNA synthetase by trapping tRNA in the editing site." science 316.5832 (2007): 1759-1761.



Tukalo, Michael, et al. "The crystal structure of leucyl-tRNA synthetase complexed with tRNALeu in the post-transfer–editing conformation." Nature structural & molecular biology 12.10 (2005): 923-930.


<br><br>

</div>
		</div>
		</body>
	</html>
