<html>
<head>
<title>Tyrosyl-tRNA Synthetase</title>
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=0.4" />
<meta name="description" content="Tyrosyl-tRNA synthetase (TyrRS) is an enzyme that attaches the amino acid tyrosine to its cognate tRNA during protein synthesis">
<meta name="keywords" content="TyrRS,YARS1,AARS Online,aarsonline,aars database,aminoacyl-tRNA synthetase,aminoacyl-tRNA ligase,AARS,database,application,tool,visualization,protein,biochemistry,molecular biology,bioinformatics">

<link rel="icon" >

<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="../../css/styles.css">

<script language='JavaScript' type='text/JavaScript' src="/js/aaRS.js"></script>
<script language='JavaScript' type='text/JavaScript' src="/js/util.js"></script>
<script type='text/javascript' src='/js/bio-pv/bio-pv.min.js'></script>

<script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>   
        
<script type="text/javascript"
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true
  }
});
</script>


 <script src="https://d3js.org/d3.v4.js"></script>
 <script  type='text/javascript' src="/js/trna.js"></script>



<script async src="https://www.googletagmanager.com/gtag/js?id=G-25G2X1FEYG"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-25G2X1FEYG');
</script>






<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>




<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS();


}




</script>

</head>
<body onload="init()">


	<div id="main">
<div class='indexMetadata'><br><br>

<!-- Content automatically pulled from readme for indexing purposes. This will be overwritten using javascript -->
<br><br>

<h1>Tyrosyl-tRNA Synthetase</h1><br><br>


Tyrosyl-tRNA synthetase (TyrRS) is an enzyme that plays a crucial role in protein synthesis by catalyzing the attachment of tyrosine to its cognate tRNA:




$ \text{Tyr} + \text{tRNA}^\text{Tyr} + \text{ATP} \xrightarrow{\text{TyrRS}} \text{Tyr-tRNA}^\text{Tyr} + \text{AMP} + \text{PP}_i  $




TyrRS shares a number of similarities with its close homolog [TrpRS](/class1/trp), which constitute subclass Ic.
TyrRS and TrpRS are homodimeric, in contrast to many Class I synthetases which operate as monomers (Gomez and Ibba, 2020).
Moreover, the two amino acids may have been among the last amino acids to emerge in the metabolism (Trifonov 2000), 
and are chemically similar (aromatic side chains), consistent with the hypotheses of the two AARS sharing a common ancestor with either dual specificity, or one synthetase spawning off the other (Doublié et al. 1995, Fournier et al. 2015).
Conserved aspartate side chains in the specificity determining helix (SDH) allow the respective enzymes to discriminate between tyrosine and tryptophan (Doublié et al. 1995).
Lastly, while most Class I synthetases approach the tRNA acceptor stem from the minor groove side, TyrRS and TrpRS alike instead approach the acceptor stem directly (de Pouplana and Schimmel 2001).




The N-terminal of TyrRS contains the Class I Rossman fold catalytic domain, presenting connecting peptide 1 (CP1) in its simplest form.
There are varying idiosyncratic domains at the C-terminus, which differ between the bacteria, archaea, and eukaryota (Bedouelle 2005).
First, tRNA recognition is achieved by the helical anticodon binding domain in bacteria and organelles (Zhang et al. 2005) and a 
distinct tRNA binding domain in the archaea and eukaryota (Wolf et al. 1999). 
The latter is homologous with the [anticodon binding domain](/superfamily/class1/Anticodon_binding_domain_WY) of TrpRS.
Second, many eukaryotic TyrRS, including the human form, contain a C-terminal RNA binding domain homologous to EMAP (Yaremchuk et al. 2002). 
This domain is also found in some forms of [MetRS](/class1/met) and [PheRS](/class2/phe2) (Wolf et al. 1999), and is the [anticodon binding domain](/superfamily/class2/Anticodon_binding_domain_DNK) of subclass IIb AARS.
Lastly, most bacterial and organellar TyrRS possess a C-terminal RNA binding domain, which belongs to the S4 superfamily (Aravind et al. 1999).


Editing activity has not been detected for TyrRS, which is one of the more promiscuous AARS with respect to amino acid recognition (Bedouelle 2005).
Coupled with the lack of cross-reactivity between TyrRS and tRNA$^\text{Tyr}$ between bacteria and archaea, this property has made TyrRS a favorable target for engineering unnatural amino acids into the genetic code (Kobayashi et al. 2003).



TyrRS is also one of several aminoacyl-tRNA synthetases expressed by the mimiviruses (Abergel et al. 2007).
In some eukaryotes, notably parasites, TyrRS exists as a double length duplicated enzyme which forms an asymmetric pseudo-dimer, with catalytic activity confined to the N-terminal paralog of the catalytic domain (Larson et al. 2011). 

<br><br>

<h2>References</h2><br><br>



Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." Rna 26.8 (2020): 910-936.






Hugues Bedouelle. "The Aminoacyl-tRNA Synthetases" CRC Press (2005): Chapter 12: Tyrosyl-tRNA Synthetases.



Abergel, Chantal, et al. "Virus-encoded aminoacyl-tRNA synthetases: structural and functional characterization of mimivirus TyrRS and MetRS." Journal of Virology 81.22 (2007): 12406-12417.




de Pouplana, Lluı́s Ribas, and Paul Schimmel. "Aminoacyl-tRNA synthetases: potential markers of genetic code development." Trends in biochemical sciences 26.10 (2001): 591-596.



Fournier, Gregory P., and E. J. Alm. "Ancestral reconstruction of a pre-LUCA aminoacyl-tRNA synthetase ancestor supports the late addition of Trp to the genetic code." Journal of molecular evolution 80 (2015): 171-185.



Doublié, Sylvie, et al. "Tryptophanyl-tRNA synthetase crystal structure reveals an unexpected homology to tyrosyl-tRNA synthetase." Structure 3.1 (1995): 17-31.



Larson, Eric T., et al. "The double-length tyrosyl-tRNA synthetase from the eukaryote Leishmania major forms an intrinsically asymmetric pseudo-dimer." Journal of molecular biology 409.2 (2011): 159-176.



Kobayashi, Takatsugu, et al. "Structural basis for orthogonal tRNA specificities of tyrosyl-tRNA synthetases for genetic code expansion." Nature Structural & Molecular Biology 10.6 (2003): 425-432.



Zhang, Yan, et al. "Crystal structures of apo wild-type M. jannaschii tyrosyl‐tRNA synthetase (TyrRS) and an engineered TyrRS specific for O-methyl-L-tyrosine." Protein science 14.5 (2005): 1340-1349.



Wolf, Yuri I., et al. "Evolution of aminoacyl-tRNA synthetases—analysis of unique domain architectures and phylogenetic trees reveals a complex history of horizontal gene transfer events." Genome research 9.8 (1999): 689-710.




Aravind, L., and Eugene V. Koonin. "Novel predicted RNA-binding domains associated with the translation machinery." Journal of molecular evolution 48 (1999): 291-302.




Yaremchuk, Anna, et al. "Class I tyrosyl-tRNA synthetase has a class II mode of cognate tRNA recognition." The EMBO Journal 21.14 (2002): 3829-3840.
<br><br>

</div>
		</div>
		</body>
	</html>
